server/test/fqhc-tests.js

```diff
@@ -248,6 +248,7 @@ describe('FQHCs', () => {
 				fqhcId: oldFQHC._id,
 			})
 			const fields = _.map(histories, 'field')
+			log.info(fields)
```

Comment:
@katelynsills Do we need this checked into master or was this just for testing?

Reply:
Whoops, we can remove it